File: progra3-0583-equipo5-frontend/Frontend_C#/SoftPac/SoftPacModel/UsuarioPaisAccesoDTO.cs
using SoftPac.Model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftPac.Model
{
    public class UsuarioPaisAccesoDTO
    {
        private UsuariosDTO usuario;
        private PaisesDTO pais;
        private Boolean acceso;

        public UsuarioPaisAccesoDTO()
        {
            this.Usuario = new UsuariosDTO();
            this.Pais = new PaisesDTO();
            Acceso = false;
        }

        public UsuarioPaisAccesoDTO(UsuariosDTO usuario, PaisesDTO pais, bool acceso)
        {
            this.Usuario = usuario;
            this.Pais = pais;
            this.Acceso = acceso;
        }

        public UsuariosDTO Usuario { get => usuario; set => usuario = value; }
        public PaisesDTO Pais { get => pais; set => pais = value; }
        public bool Acceso { get => acceso; set => acceso = value; }
    }
}

File: BackEnd/Backend_Java/SoftPacModel/src/main/java/pe/edu/pucp/softpac/model/UsuarioPaisAccesoDTO.java
package pe.edu.pucp.softpac.model;

public class UsuarioPaisAccesoDTO {
    private UsuariosDTO usuario;
    private PaisesDTO pais;
    private Boolean acceso;

    public UsuarioPaisAccesoDTO(){
        setUsuario(null);
        setPais(null);
        setAcceso(false);
    }
    public UsuarioPaisAccesoDTO(UsuariosDTO usuario, PaisesDTO pais){
        setUsuario(usuario);
        setPais(pais);
        setAcceso(true);
    }

    public PaisesDTO getPais() {
        return pais;
    }

    public void setPais(PaisesDTO pais) {
        this.pais = pais;
    }

    public Boolean getAcceso() {
        return acceso;
    }

    public void setAcceso(Boolean acceso) {
        this.acceso = acceso;
    }

    public UsuariosDTO getUsuario() {
        return usuario;
    }

    public void setUsuario(UsuariosDTO usuario) {
        this.usuario = usuario;
    }
}
